src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -27,4 +27,30 @@ public Integer getLength() {
     public String getColor() {
         return color;
     }
+
+    @Override
+    public boolean equals(Object object) {
+        if (this == object) {
+            return true;
+        }
+        if (object == null || !object.getClass().equals(Rectangle.class)) {
+            return false;
+        }
+        Rectangle rectangle = (Rectangle) object;
+        return (width == null && rectangle.width == null)
```

Comment:
just a note: this can be simplified
```suggestion
        return (width == rectangle.width
```

Reply:
done